aws/resource_aws_elb_test.go

```diff
@@ -1231,6 +1231,11 @@ resource "aws_elb" "foo" {
     lb_protocol = "http"
   }
 }
+
+# See https://github.com/terraform-providers/terraform-provider-aws/issues/2498
```

Comment:
Did you mean to include these outputs in the test configurations? Only other place I see one is: https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_emr_cluster_test.go#L636-L638

Reply:
Yes, that was the intention - to reference it _somewhere_ to make the test fail if someone gets the idea of re-implementing #2314 in the future 😄